test/controllers/update/update.any.trusts.involved.spec.ts

```diff
@@ -0,0 +1,105 @@
+jest.mock('../../../src/middleware/company.authentication.middleware');
+jest.mock('../../../src/utils/application.data');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/middleware/navigation/update/has.overseas.entity.middleware');
+
+import { NextFunction, Request, Response } from "express";
+import { beforeEach, expect, jest, test, describe } from "@jest/globals";
+import request from "supertest";
+import {
+  UPDATE_ANY_TRUSTS_INVOLVED_PAGE,
+  UPDATE_ANY_TRUSTS_INVOLVED_URL,
+  SECURE_UPDATE_FILTER_PAGE,
+  UPDATE_INTERRUPT_CARD_PAGE,
+  UPDATE_TRUSTS_SUBMIT_BY_PAPER_URL,
+  UPDATE_INTERRUPT_CARD_URL
+} from "../../../src/config";
+import app from "../../../src/app";
+import {
+  PAGE_TITLE_ERROR,
+  UPDATE_ANY_TRUSTS_INVOLVED_HEADING,
+  ANY_MESSAGE_ERROR,
+  SERVICE_UNAVAILABLE,
+  RADIO_BUTTON_YES_SELECTED,
+  RADIO_BUTTON_NO_SELECTED
+} from "../../__mocks__/text.mock";
+import { ErrorMessages } from '../../../src/validation/error.messages';
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+import { logger } from "../../../src/utils/logger";
+import { AnyTrustsInvolvedKey, yesNoResponse } from "../../../src/model/data.types.model";
+
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockLoggerDebugRequest = logger.debugRequest as jest.Mock;
+
+describe("Update any trusts involved controller tests", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("GET tests", () => {
+    test(`renders the ${UPDATE_ANY_TRUSTS_INVOLVED_PAGE} page`, async () => {
+      const resp = await request(app).get(UPDATE_ANY_TRUSTS_INVOLVED_URL);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(UPDATE_ANY_TRUSTS_INVOLVED_HEADING);
+      expect(resp.text).not.toContain(RADIO_BUTTON_YES_SELECTED);
+      expect(resp.text).not.toContain(RADIO_BUTTON_NO_SELECTED);
+      expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+    });
+
+    test("catch error when rendering the page", async () => {
+      mockLoggerDebugRequest.mockImplementationOnce( () => { throw new Error(ANY_MESSAGE_ERROR); });
+      const resp = await request(app).get(UPDATE_ANY_TRUSTS_INVOLVED_URL);
+
+      expect(resp.status).toEqual(500);
+      expect(resp.text).toContain(SERVICE_UNAVAILABLE);
+    });
+  });
+
+  describe("POST tests", () => {
+    test(`redirect to ${SECURE_UPDATE_FILTER_PAGE} page when ${yesNoResponse.Yes} is selected`, async () => {
```

Comment:
Think Cory left a similar comment somewhere, let's not use a variable name here so we can run the tests individually